theano/gpuarray/opt.py

```diff
@@ -1967,6 +1967,16 @@ def local_gpu_solve(op, context_name, inputs, outputs):
         return
     return GpuCusolverSolve()
 
+
+# Cholesky decomposition
+@register_opt('fast_compile')
+@op_lifter([slinalg.Cholesky])
+@register_opt2([theano.tensor.slinalg.Cholesky], 'fast_compile')
+def local_gpu_cholesky(op, context_name, inputs, outputs):
+    if not cusolver_available:
+        return
+    return GpuCholesky()
```

Comment:
`slinalg.Cholesky` contains two attributes `lower` and `destructive` (which can be interpreted as `inplace`), so transfer them also into `GpuCholesky` here:
```
return GpuCholesky(lower=op.lower, inplace=op.destructive)
```